priceCalculator.js

```diff
@@ -0,0 +1,17 @@
+
+class PriceCalculator{
+
+     constructor(crust, size, topping){
+        this.topping = topping;
+        this.crust = crust;
+        this.size = size;
+     }
+
+     calculate(){
+        
+        var totalPrice = (this.topping + this.crust + this.size ) * 1.12;
+        return totalPrice;
+     }
+}
+
+exports.PriceCalculator = PriceCalculator;
```

Comment:
we were making changes to the wrong PriceCalculator.js (the one inside the router xD)
I removed the duplicated file.

also, the exports was wrong, we were using:
**module.exports = PriceCalculator;** 
and it should be:
**exports.PriceCalculator = PriceCalculator;**